src/Components/Modal/Modal.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import Modal, { ModalProps } from "react-native-modal";
+import { isWeb } from "src/Helpers/platform";
+import WebModal from "./WebModal";
+
+const HybridModal = (props: ModalProps) => {
```

Comment:
use `PureComponent` here, to make the component to do a shallow comparison of the props.